project_euler/problem_050/sol1.py

```diff
@@ -0,0 +1,85 @@
+"""
+Problem 50: https://projecteuler.net/problem=50
+
+The prime 41, can be written as the sum of six consecutive primes:
+
+41 = 2 + 3 + 5 + 7 + 11 + 13
+This is the longest sum of consecutive primes that adds to a prime below one-hundred.
+
+The longest sum of consecutive primes below one-thousand that adds to a prime, contains 21 terms, and is equal to 953.
+
+Which prime, below one-million, can be written as the sum of the most consecutive primes?
+"""
+
+
+def get_primes(limit: int) -> (set, list):
+    """
+    Returns the set & list(ordered) of primes up to the below limit
+    using Sieve of Eratosthenes
+
+    >> get_primes(10)
+    ({2, 3, 5, 7}, [2, 3, 5, 7])
+    """
+    SV_MAX = limit
```

Comment:
You can directly use the `limit` parameter.

Reply:
fixed